### PR TITLE
add Get topics api

### PR DESCRIPTION
**Pull request by Liam-Tomlinson**
This is task 2.  An api has been create that gets the topic table.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      return db.end();
+    });
+
+
+describe('GET /api/topics', () => {
+    test('status:200, responds wth correct status', () => {
+        return request(app)
+          .get('/api/topics')
+          .expect(200);    
+    })
```
Comment (nc-christian-dutton) [inline __tests__/app.test.js:20]: You should have a .then here (or a separate test) that checks what the topics array looks like. If you access body.data (you have sent over an object in the controller with a key of data, which will hold the topics array), you can then see what is being returned. 

To test this, I would advise a forEach over the array and you can `expect(typeof slug).toBe('string')` - or something along those lines - for every property within the topics array. I would also check the length of the topics array to make sure you are getting them all back!

File: controllers/topics.controllers.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+};
+
+
+
+
+
+exports.sendSnacks = (req, res) => {
+    selectSnacks().then((snacks) => res.status(200).send({ snacks }));
+  };
```
Comment (nc-christian-dutton) [inline controllers/topics.controllers.js:15]: Don't forget to remove any code that isn't a part of the project/ticket ☝️

**Comment by nc-christian-dutton** [comment]
MVC and general set up is great - you should just make that test more robust!

File: controllers/topics.controllers.js (hunk 2 above)
Comment (Liam-Tomlinson) [reply controllers/topics.controllers.js:15]: Oh yeah, sorry Christian was using this as a template from a previous project it is now removed.

File: __tests__/app.test.js (hunk 1 above)
Comment (Liam-Tomlinson) [reply __tests__/app.test.js:20]: Thanks for the advice, I have added another test which tests for the data type and the length of the test data.

**Pull request merged**